assets/locales/active.cy.toml

```diff
@@ -462,6 +462,77 @@ few = "Related links"
 many = "Related links"
 other = "Related links"
 
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# Current dataset page
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+[DatasetPageTitle]
+description = "Dataset(s)"
+one = "Dataset(s)"
+
+[IsAccreditedStatistic]
+description = "This is an accredited National Statistic. Click for information about types of official statistics."
+one = "This is an accredited National Statistic. Click for information about types of official statistics."
+
+[DatasetIDHelp]
+description = "This series ID is a unique random identifier for this individual time series. The digits themselves do not have any specific meaning"
+one = "This series ID is a unique random identifier for this individual time series. The digits themselves do not have any specific meaning"
+
+[WhatIsThis]
+description = "What is this"
+one = "What is this?"
+
+[ReturnTo]
+description = "Return to"
+one = "Return to"
+
+[FilesOverview]
+description = "Provides files to download data as it existed for this dataset on previous dates."
+one = "Provides files to download data as it existed for this dataset on previous dates."
+
+[RevisionIntro]
+description = "Statistics are most often revised for 1 of 2 reasons:"
+one = "Statistics are most often revised for 1 of 2 reasons:"
+
+[RevisionReasonOne]
+description = "For certain statistics initial estimates are released with the expectation that these may be revised and updated as further data becomes available."
+one = "For certain statistics initial estimates are released with the expectation that these may be revised and updated as further data becomes available."
+
+[RevisionReasonTwo]
+description = "Revisions may also be made when methods or systems are changed."
+one = "Revisions may also be made when methods or systems are changed."
+
+[PlannedRevisionOverview]
+description = "These types of planned revisions should not be confused with errors in released statistics, which are genuine mistakes. Such mistakes occur rarely and, when they do happen, corrections are made in a timely manner, announced and clearly explained to users in line with the Code of Practice for Official Statistics (Principle 2, Practice 7)."
+one = "These types of planned revisions should not be confused with errors in released statistics, which are genuine mistakes. Such mistakes occur rarely and, when they do happen, corrections are made in a timely manner, announced and clearly explained to users in line with the Code of Practice for Official Statistics (Principle 2, Practice 7)."
+
+[LatestVersionHeading]
+description = "Latest version"
+one = "Latest version"
+
+[PreviousVersionsHeading]
+description = "Previous versions"
+one = "Previous versions"
```

Comment:
Confused as to how Related Links is an example on how to make use of plurals

`[RelatedLinks]
description = "Related links"
one = "Related links"
two = "Related links"
few = "Related links"
many = "Related links"
other = "Related links"`

Or should one actually = "Related Link"?

Reply:
Related links might not have been the best example after all. Seems that one has been butchered a little. As you point out yes 'one' should be 'Related Link' rather than with the (s).

Maybe SaveChanges is a better example
The English file has:
```toml
[SaveChanges]
description = "Save changes"
one = "Save change"
other = "Save changes"
```

Whereas the Welsh has:
```toml
[SaveChanges]
description = "Save changes"
one = "Save change"
two = "Save changes"
few = "Save changes"
other = "Save changes"
```

Then to use 'other' (in English and Welsh) we can use a number like 4 `{{ localise "SaveChanges" .Language 4}}`

But if in doubt worth checking the package and documentation https://github.com/nicksnyder/go-i18n

Plural rules can be found here per a language: http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html